### PR TITLE
fix a bug

### PR DESCRIPTION
**Pull request by puredev55**
i deleted the javascript command

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +0,0 @@
-Alert("checkofassa")
```
Comment (ykbeili) [inline index.js:1]: Wa7esh 7akiki ya bo salih.

**Review by ykbeili — APPROVED** [review]

**Comment by puredev55** [comment]
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="style.css">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="
***@***.***,100;1,600&family=Zen+Dots&display=swap"
rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="
***@***.***&family=Zen+Dots&display=swap"
rel="stylesheet"> <link href="
https://fonts.googleapis.com/css2?family=Zen+Dots&display=swap"
rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="
***@***.***,100;1,600&family=Zen+Dots&display=swap"
rel="stylesheet">
    <link rel="stylesheet" href="
***@***.***/dist/css/bootstrap.min.css"
integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
crossorigin="anonymous">
  </head>
  <body>
<nav  class="navbar navbar-dark navbar-expand-sm sticky-top" id="navy">
  <a href="" class="navbar-brand">HomeTech</a>
  <button class="navbar-toggler" data-toggle="collapse"
data-target="#mynav">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse" id="mynav">
    <ul class="navbar-nav ml-auto">
      <li class="nav-item">
        <a href="" class="nav-link">Home</a>
      </li>
      <li   class="nav-item dropdown">
        <a href="#" class="nav-link dropdown-toggle" id="drp" role="button"
data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
>Phones</a>
      <div class="dropdown-menu"  aria-labelledby="drp">
        <a href="#"  class="dropdown-item">Phone-brand</a>
        <a href="#" class="dropdown-item">Phone-brand</a>
        <a href="#" class="dropdown-item">Phone-brand</a>
        <a href="#" class="dropdown-item">Phone-brand</a>
      </div>
      </li>
      <li class="nav-item">
        <a href="" class="nav-link">PC</a>
      </li>
      <li class="nav-item">
        <a href="" class="nav-link">Accessories</a>
      </li>
      <li class="nav-item">
        <a href="" class="nav-link">About</a>
      </li>
      
<button type="button" class="btn btn-primary" data-toggle="modal"
data-target="#exampleModal">
  Launch demo modal
</button>


<div class="modal fade" id="exampleModal" tabindex="-1"
aria-labelledby="exampleModalLabel" aria-hidden="true">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
        <button type="button" class="close" data-dismiss="modal"
aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        ...
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary"
data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </div>
    </div>
  </div>
</div>
    </ul>
    </div>
</nav>

</body>
</html>
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
crossorigin="anonymous"></script>
<script src="
***@***.***/dist/umd/popper.min.js"
integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
crossorigin="anonymous"></script>
<script src="
***@***.***/dist/js/bootstrap.min.js"
integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF"
crossorigin="anonymous"></script></body>


‫في الأحد، 24 يناير 2021 في 12:49 م تمت كتابة ما يلي بواسطة ‪Yousef
Kbeili‬‏ ***@***.***‬‏>:‬

**Comment by puredev55** [comment]
‫في الأحد، 25 أبريل 2021 في 4:47 م تمت كتابة ما يلي بواسطة ‪pure market‬‏ <‪
***@***.***‬‏>:‬

**Comment by ykbeili** [comment]
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="
***@***.***/dist/css/bootstrap.min.css"
integrity=
"sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
crossorigin="anonymous">
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity=
"sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
crossorigin="anonymous"></script>
<script src="
***@***.***/dist/js/bootstrap.bundle.min.js
" integrity=
"sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"
crossorigin="anonymous"></script>
</head>
<body>
<nav class="navbar navbar-dark navbar-expand-sm sticky-top" id="navy">
<a href="" class="navbar-brand">HomeTech</a>
<button class="navbar-toggler" data-toggle="collapse" data-target="#mynav">
<span class="navbar-toggler-icon"></span>
</button>
<div class="collapse navbar-collapse" id="mynav">
<ul class="navbar-nav ml-auto">
<li class="nav-item">
<a href="" class="nav-link">Home</a>
</li>
<li class="nav-item dropdown">
<a href="#" class="nav-link dropdown-toggle" id="drp" role="button"
data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Phones</a>
<div class="dropdown-menu" aria-labelledby="drp">
<a href="#" class="dropdown-item">Phone-brand</a>
<a href="#" class="dropdown-item">Phone-brand</a>
<a href="#" class="dropdown-item">Phone-brand</a>
<a href="#" class="dropdown-item">Phone-brand</a>
</div>
</li>
<li class="nav-item">
<a href="" class="nav-link">PC</a>
</li>
<li class="nav-item">
<a href="" class="nav-link">Accessories</a>
</li>
<li class="nav-item">
<a href="" class="nav-link">About</a>
</li>

<button type="button" class="btn btn-primary" data-toggle="modal"
data-target="#exampleModal">
Launch demo modal
</button>

</ul>
</div>
</nav>
<div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby=
"exampleModalLabel" aria-hidden="true">
<div class="modal-dialog">
<div class="modal-content">
<div class="modal-header">
<h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
<button type="button" class="close" data-dismiss="modal" aria-label="Close">
<span aria-hidden="true">&times;</span>
</button>
</div>
<div class="modal-body">
...
</div>
<div class="modal-footer">
<button type="button" class="btn btn-secondary" data-dismiss="modal">Close</
button>
<button type="button" class="btn btn-primary">Save changes</button>
</div>
</div>
</div>
</div>
</body>
</html>

**Comment by ykbeili** [comment]
You just need to add the modal outside the nav closing tag (not before
</nav>) because of the default z-index for the navbar.
I fixed the indentation for you as well.
Let me know if you have any questions.
Cheer 7abibona al lazzam.

**Comment by ykbeili** [comment]
Cheers*

**Comment by puredev55** [comment]
thanks maaaan

‫في الأحد، 25 أبريل 2021 في 7:24 م تمت كتابة ما يلي بواسطة ‪Yousef Kbeili‬‏
***@***.***‬‏>:‬

> Cheers*
>

**Pull request merged**